components/BurgerMenuNavItems.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <ul class="burger-menu-nav__items">
```

Comment:
Pour suivre le nom du fichier:

```suggestion
  <ul class="burger-menu-nav-items">
```